openapi/openapi.yml

```
@@ -58,4 +58,66 @@ paths:
                 properties:
                   message:
                     type: string
-                    example: "Cannot set table count higher than 20"
\ No newline at end of file
+                    example: "Cannot set table count higher than 20"
+  /setting/items:
+    post:
+      summary: "add new item"
+      requestBody:
+        content:
+          multipart/form-data:
+            schema:
+              type: object
+              required:
+                - name
+                - price
+                - image
+                - cost_price
+                - tax_rate_id
+              properties:
+                name:
+                  type: string
+                price:
+                  type: number
```

商品価格だと思っていて、

```suggestion
                  type: integer
```

でどうでしょうか。
numberですと、1.2などの小数点もOKになってしまうので
https://swagger.io/docs/specification/data-models/data-types/